lib/screens/aim/text_aim.dart

```diff
@@ -0,0 +1,9 @@
+class TextAim {
+  static String getAim() {
```

Comment:
🔴 
these should be in `strings.dart`, as all string constants are supposed to be.